includes/shortcodes/class-evf-shortcode-form.php

```diff
@@ -514,6 +514,43 @@ private static function get_field_properties( $field, $form_data, $attributes =
 		$form_id  = absint( $form_data['id'] );
 		$field_id = sanitize_text_field( $field['id'] );
 
+		// Data to embed on the field container
+		$container_data = array();
+
+		// Embed required-field-message to the container if the field is required
+		if ( isset( $field['required'] ) && ( '1' === $field['required'] || true === $field['required'] ) ) {
+			$has_sub_fields = false;
+			$sub_field_messages = array();
+			if ( 'likert' === $field['type'] ) {
+				$has_sub_fields = true;
+				$likert_rows    = isset( $field['likert_rows'] ) ? $field['likert_rows'] : array();
+				$row_keys       = array();
+				foreach ( $likert_rows as $row_key => $row_label ) {
+					$row_keys[] = $row_key;
+					$row_slug   = 'required-field-message-' . $row_key;
+					$sub_field_messages[ $row_key ] = isset( $field[ $row_slug ] ) ? $field[ $row_slug ] : '';
+				}
+				$container_data[ 'row-keys' ] = json_encode( $row_keys );
+			} else if ( 'address' === $field['type'] ) {
+				$has_sub_fields = true;
+				// Format : 'sub_field_type' => 'error_message'
```

Comment:
Unwanted codes must be removed. Please remove all other if there exists.

Reply:
Removed the commented code.